src/main/java/seedu/address/model/policy/PolicyDate.java

```diff
@@ -22,6 +22,8 @@ public class PolicyDate {
     public static final String VALIDATION_DATE_FORMAT = "dd-MM-yyyy";
     public static final String DEFAULT_VALUE = "01-01-1000";
 
+    public static final String DEFAULT_COMPARISON_VALUE = "1000-01-01";
+
```

Comment:
Why DEFAULT_COMPARISON_VALUE is in YYYY-MM-DD? I think it should follow the dd-MM-yyyy format, no?